### ISSUE TITLE
Brave browser blocking website's video content

### ISSUE THREAD

**Issue by SyedSan1** [brave/brave-browser#23145]
## Description 

We provide a video technology platform for partners to embed social media like videos that are likable, comment-able and purchasable on customer's website.
Many customers pointed out that their video is getting blocked by Brave.

Note, we don't put outside advertisements. We provide technology that the customers can use to put their product videos which is what happens in 90%+ cases. Occasionally, a customer may decide to insert ads.

We recently worked with the AdGuard team to get this issue resolved here by allowing filters to pass our content CDNs.
https://github.com/AdguardTeam/AdguardFilters/issues/111021

Similarly, we worked with uBlockOrigin and EasyList team to get the issue resolved here
https://github.com/uBlockOrigin/uAssets/issues/12153

We'd appreciate a similar resolution on this. Thank you.

## Steps to Reproduce

   1. Please try any of the merchants who are using Firework provided application to run video or livestream on their website. Here are a few examples:
   https://www.elizabetharden.com/live-stream.html
   https://scripbox.com/short-stories/
   https://www.marieclaire.com.au/

   2. 
   3. 

## Actual result: 

All videos get blocked even though these are not ads. These are the video content that the website owner is publishing on our Firework video saas application.

## Expected result:
Native videos or livestreams should be allowed to run and not blocked by the Brave browser. I believe Brave filter presumes that the videos are ads but they're native short form videos about product or customer reviews.

## Reproduces how often: 

Just visit any of the sites I mentioned above to reproduce the issue.

## Version/Channel Information:

- Can you reproduce this issue with the current release?
- Can you reproduce this issue with the beta channel?
- Can you reproduce this issue with the dev channel?
- Can you reproduce this issue with the nightly channel?

I am not sure what's meant here, but yes this used to happen with other adblockers such as Easylist, AdBlock etc. and we work with them to have it unblocked since these are native content.

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields?
- Does the issue resolve itself when disabling Brave Rewards?
- Is the issue reproducible on the latest version of Chrome?

**Comment by rebron** [comment]
cc: @ryanbr

**Comment by ryanbr** [comment]
Previously discussed @SyedSan1 in https://github.com/uBlockOrigin/uAssets/issues/12153

We haven't added or changed any blocks related to fireworks since the [last commit ](https://github.com/easylist/easylist/commit/caac39ee564de5e1760c5ed76541c856d184a68a)

**Comment by SyedSan1** [comment]
Thank you @ryanbr and @rebron 
Raised the issue here. https://github.com/uBlockOrigin/uAssets/issues/13512

Question - you mentioned that Brave is implementing the standard uBO filterlists, and uBO had already unblocked the fwcdn2, then why Brave is blocking it. Is there another filterlist that Brave is using besides uBO?

**Comment by ryanbr** [comment]
Closing ticket, needs to be addressed (if that's the decision) by `Peter Lowe’s Ad and tracking server list`

**Comment by richardvaldivieso** [comment]
Good morning. I will piggyback on this issue and see if it is worth opening again. Yesterday SpringOne release its learning academy. Spring.Academy. I registered, but I noticed that all the videos did not show. I checked the console and the video URL showed a 403 message. (Failed to load resource: the server responded with a status of 403 (Forbidden)). I have tried Google Chrome and login with the same credentials, and the video play without problems. 

Here is my Brave browser version: [Version 1.47.171 Chromium: 109.0.5414.87 (Official Build) (64-bit)](https://brave.com/latest/)
Running on Windows 11.
